aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,47 +1,26 @@
 ---
 remote_endpoint: ~
-name: "v1.11"
+name: "v1.12"
 proposals:
   - name: step_1_increase_max_txn_gas
     metadata:
       title: "Increase max txn gas temporarily for framework upgrade"
       description: "Increase max txn gas temporarily for framework upgrade"
     execution_mode: MultiStep
     update_sequence:
-      - DefaultGasWithOverrideOld:
```

Comment:
have we updated the framework upgrade test baseline from 1.10.x to 1.11? otherwise this is still needed.

Reply:
@zjma , the base node version is bumped up in this PR. Can you see if that looks ok to you?

Reply:
looks good. i'm just not sure this needs to be here or in the target branch to make sure the framework upgrade test pick it up